src/packages/console/src/pages/homePage.tsx

```diff
@@ -122,53 +124,57 @@ export const HomePage = (): React.ReactElement => {
-              <Paper key={index} elevation={0} className={classes.accountBox}>
-                <Box width={1} display='flex' justifyContent='start' alignItems='baseline'>
-                  <Typography variant='h5' className={classes.accountName}>
+              <Box variant='bordered'>
+                {/* <Paper key={index} elevation={0} className={classes.accountBox}> */}
+                {/* <Box width={1} display='flex' justifyContent='start' alignItems='baseline'> */}
+                <Stack direction={Direction.Horizontal} contentAlignment={Alignment.Start} childAlignment={Alignment.Center}>
+                  <Text variant='header5'>
                     {account.name}
-                  </Typography>
-                  <Typography color='textSecondary' className={classes.accountType}>{`(${account.accountType})`}</Typography>
-                  {authManager.getHasJwtPermission(`acc-${account.accountId}-ed`) && <Button color='primary' onClick={(): void => onManageAccountClicked(account)}>Manage</Button>}
-                  <Box flexGrow={1} />
-                  {authManager.getHasJwtPermission(`acc-${account.accountId}-adm`) && <Button color='primary' onClick={(): void => onCreateSiteClicked(account)}>Create site</Button>}
-                </Box>
-                <Grid container spacing={2} className={classes.siteCardGrid}>
-                  {accountSites[account.accountId].map((site: Site, innerIndex: number): React.ReactElement => (
-                    <Grid item xs={12} sm={6} md={4} lg={3} key={innerIndex}>
-                      <SiteCard site={site} onSiteClicked={onSiteClicked} isEnabled={authManager.getHasJwtPermission(`st-${site.siteId}-vw`)} />
-                    </Grid>
-                  ))}
-                  {accountSites[account.accountId].length === 0 && (
-                    <Grid item xs={12}>
-                      <Typography color='textSecondary'>
-                        {'No sites yet. Create one now!'}
-                      </Typography>
-                    </Grid>
-                  )}
-                </Grid>
-              </Paper>
+                  </Text>
+                  <Text variant='colored'>{`(${account.accountType})`}</Text>
+                  {authManager.getHasJwtPermission(`acc-${account.accountId}-ed`) && <Button onClicked={(): void => onManageAccountClicked(account)} text='Manage' />}
+                  <Stack.Item growthFactor={1} shrinkFactor={1} />
+                  {authManager.getHasJwtPermission(`acc-${account.accountId}-adm`) && <Button onClicked={(): void => onCreateSiteClicked(account)} text='Create site' />}
+                </Stack>
+                {/* </Box> */}
+                <Spacing variant={PaddingSize.Wide} />
+                {accountSites[account.accountId].map((site: Site, innerIndex: number) => (
+                  <EqualGrid contentAlignment={Alignment.Fill} shouldAddGutters={true}>
```

Comment:
EqualGrid should be outside the map, there should only be one of these created, not one per site.